xml/System.Net.Http/HttpRequestOptions.xml

```diff
@@ -109,8 +109,8 @@
         <Parameter Name="item" Type="System.Collections.Generic.KeyValuePair&lt;System.String,System.Object&gt;" />
       </Parameters>
       <Docs>
-        <param name="item">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="item">The object to add to the <see cref="T:System.Collections.Generic.ICollection`1" />.</param>
+        <summary>Adds an item to the <see cref="T:System.Collections.Generic.ICollection`1" />.</summary>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Net.Http.HttpRequestOptions> instance is cast to an <xref:System.Collections.Generic.ICollection`1> interface.

          ]]></format>
        </remarks>
```